amalgamation/Makefile

```diff
@@ -12,14 +12,12 @@ endif
 
 ifndef ANDROID
     export ANDROID=0
-else
-	DEFS+=-DMSHADOW_USE_SSE=0
```

Comment:
this logic is handled. it is intention that don't put defines into the cc file since the order of the file is not fixed and may cause issue.